src/test/tests/graphics/scenery/tests/unit/CurveTests.kt

```diff
@@ -1,21 +1,22 @@
 package graphics.scenery.tests.unit
 
+import org.joml.*
 import graphics.scenery.numerics.Random
-import graphics.scenery.CatmullRomSpline
-import graphics.scenery.Curve
-import graphics.scenery.UniformBSpline
+import graphics.scenery.*
 import graphics.scenery.utils.LazyLogger
-import org.joml.Vector3f
 import org.junit.Test
 import org.lwjgl.BufferUtils
+import kotlin.reflect.full.declaredMemberFunctions
+import kotlin.reflect.jvm.isAccessible
```

Comment:
These do not seem to be used anywhere, make sure you clean up imports.

Reply:
I believe they are necessary for the reflection - I access private functions in this test.